crates/lakekeeper/src/service/tasks/tabular_expiration_queue.rs

```diff
@@ -115,15 +129,17 @@ async fn instrumented_expire<C: CatalogStore, A: Authorizer>(
     let entity_id = task.task_metadata.entity_id;
     match handle_table::<C, A>(catalog_state.clone(), authorizer, task).await {
         Ok(()) => {
-            tracing::debug!("Task of `{QN_STR}` worker exited successfully. {entity_id} deleted.");
+            tracing::debug!(
+                "Task of `{QN_STR}` worker exited successfully. {entity_id:?} deleted."
```

Comment:
Don't use debug prints in logs / traces eccept if you know there is a good representation (i.e. Vec is OK).
This option struct is not readable.